lottie/meson.build

```diff
@@ -15,6 +15,8 @@ source_file = [
 
 thorvg_lib_dep = declare_dependency(dependencies: thorvg_lib, include_directories: headers)
 
+add_project_link_arguments('-fopenmp', language: 'cpp')
```

Comment:
could you please try with in `thorvg/src/renderer/sw_engine/meson.build`
```
//omp_dep = dependency('openmp', required: false)
//    if (omp_dep.found())                                                                                                                             //sw_compiler_args = '-DTHORVG_SW_OPENMP_SUPPORT=1'                                                                                        omp_dep = declare_dependency(link_args: ['-fopenmp'])
```

Reply:
Doesn't work :(